.github/workflows/main.yml

```diff
@@ -22,19 +22,32 @@ jobs:
         run: |
           sudo mkdir /opt/local.old
           sudo mv /usr/local/* /opt/local.old
```

Comment:
Is removing everything from /usr/local going to be correct on a Linux system? We do it here because the macOS runners GitHub Actions provides come pre-installed with Homebrew in /usr/local and we don't want that.

Reply:
Yes you're right, it's not required to remove /usr/local/ on the ubuntu runner. Just having a quick flick through the [pre-installed software](https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md#package-management), Homebrew is installed. However, it's in `/home/linuxbrew` and not in PATH by default.

I've changed it so that it's only removed on macOS.